queries/hcl/highlights.scm

```diff
@@ -94,6 +94,6 @@
 ; var.foo, data.bar
 ;
 ; first element in get_attr is a keyword or a reference to a keyword
-(expression (variable_expr (identifier) @keyword) (get_attr (identifier) @field))
```

Comment:
if this change is necessary, then the comment explaining the keyword needs to be updated.

Reply:
Ah yes - thanks for catching that. Fixed.